### PR TITLE
feat(billing.autorenew): My services improvements 

### PR DESCRIPTION
**Pull request by marie-j**
### Description of the change 

- Rename "Etat" to "Disponibilité"
- Add filter on "Disponibilité" column

### Related 
- [ ] 2API 
- [ ] https://github.com/ovh-ux/manager/pull/1468

MBE-397, MANAGER-3264

### CODE REVIEW

**Review by antleblanc — APPROVED** [review]

File: client/app/models/BillingService.class.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ export default class BillingService {
     Object.assign(this, service);
 
     this.expirationDate = moment(this.expiration);
+    this.state = this.isSuspended() ? 'EXPIRED' : 'UP';
```
Comment (FredericEspiau) [inline client/app/models/BillingService.class.js:9]: This is weird. Why can I `service.isSuspended()` but I can't `service.isUp()` ?

File: client/app/billing/autoRenew/autorenew.service.js [hunk 2]
```diff
@@ -77,6 +84,13 @@ export default class {
     }), {});
   }
 
+  getStatesTypes() {
```
Comment (FredericEspiau) [inline client/app/billing/autoRenew/autorenew.service.js:86]:
```suggestion
  getStateTypes() {
```
but this is nitpicking :)

**Comment by jleveugle** [comment]
Hello @marie-j,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Review by antleblanc — CHANGES_REQUESTED** [review]
@marie-j Could you please resolve existing conflict?

Thanks!

File: client/app/models/BillingService.class.js (hunk 1 above)
Comment (marie-j) [reply client/app/models/BillingService.class.js:9]: Could do but `serviceisUp() === !service.isSuspended()`.

**Review by JeremyDec — DISMISSED** [review]

**Comment by marie-j** [comment]
Closing this as it will be handled by https://github.com/ovh/manager/pull/1639

**Comment by antleblanc** [comment]
@marie-j Could you please delete the branch?
Thanks!

**Pull request closed without merging**